helpers/stringHelper.js

```diff
@@ -1,48 +1,62 @@
 export const upperFirst = (str) => {
-  if (!str) return str;
-  return str.charAt(0).toUpperCase() + str.slice(1);
-};
+  if (!str) return str
+  return str.charAt(0).toUpperCase() + str.slice(1)
+}
 
 export const trimObject = (obj) => {
-  const newObj = JSON.parse(JSON.stringify(obj));
+  const newObj = JSON.parse(JSON.stringify(obj))
   Object.keys(newObj).forEach((key) => {
     if (typeof obj[key] === 'string') {
-      obj[key] = obj[key].trim();
+      obj[key] = obj[key].trim()
     }
-  });
-  return newObj;
-};
+  })
+  return newObj
+}
 
 export const removeWhiteSpaces = (str) => {
   if (!str) {
-    return '';
+    return ''
   }
-  return str.replace(/\s/g, '');
-};
+  return str.replace(/\s/g, '')
+}
 
 export const camelToSentence = (text) => {
-  const result = text.replace(/([A-Z])/g, ' $1');
-  return result.charAt(0).toUpperCase() + result.slice(1);
-};
+  const result = text.replace(/([A-Z])/g, ' $1')
+  return result.charAt(0).toUpperCase() + result.slice(1)
+}
 
 export function kebabToCamel(kebabCase) {
   if (!kebabCase || kebabCase == '') {
-    return '';
+    return ''
   }
-  return kebabCase.replace(/-([a-z])/g, (match, letter) =>
-    letter.toUpperCase(),
-  );
+  return kebabCase.replace(/-([a-z])/g, (match, letter) => letter.toUpperCase())
 }
 
 export function camelToKebab(camelCase) {
   if (!camelCase || camelCase == '') {
-    return '';
+    return ''
   }
-  return camelCase.replace(/([a-z0-9])([A-Z])/g, '$1-$2').toLowerCase();
+  return camelCase.replace(/([a-z0-9])([A-Z])/g, '$1-$2').toLowerCase()
 }
 
 export function toTitleCase(str) {
   return str.replace(/\w\S*/g, function (txt) {
-    return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();
-  });
+    return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase()
+  })
+}
+
+export const htmlToPlainText = (html) => {
+  if (!html) return ''
+
+  const tempDiv = document.createElement('div')
+  tempDiv.innerHTML = html
+
+  const textWithBreaks = tempDiv.innerHTML
+    .replace(/<\/p>/gi, '\n')
+    .replace(/<\/h[1-6]>/gi, '\n')
+    .replace(/<br\s*\/?>/gi, '\n')
+    .replace(/<\/div>/gi, '\n')
+
+  tempDiv.innerHTML = textWithBreaks
+  return tempDiv.textContent || tempDiv.innerText || ''
 }
```

Comment:
Prolly safer to just use what millions of others are already using? 

https://www.npmjs.com/package/string-strip-html
https://www.npmjs.com/package/striptags